test/requests/baremetal/chassis_tests.rb

```diff
@@ -0,0 +1,61 @@
+require "test_helper"
+
+describe "Fog::Baremetal[:openstack] | Baremetal chassis requests" do
+  before do
+    @baremetal = Fog::Baremetal[:openstack]
+
+    @chassis_format = {
+      'description' => String,
+      'uuid'        => String,
+      'links'       => Array
+    }
+
+    @detailed_chassis_format = {
+      'description' => String,
+      'uuid'        => String,
+      'created_at'  => String,
+      'updated_at'  => Fog::Nullable::String,
+      'extra'       => Hash,
+      'nodes'       => Array,
+      'links'       => Array
+    }
+  end
+
+  describe "success" do
+    it "#list_chassis" do
+      @baremetal.list_chassis.body.must_match_schema('chassis' => [@chassis_format])
+    end
+
+    it "#list_chassis" do
+      @baremetal.list_chassis.body.must_match_schema('chassis' => [@chassis_format])
+    end
+
+    it "#list_chassis_detailed" do
+      @baremetal.list_chassis_detailed.body.must_match_schema('chassis' => [@detailed_chassis_format])
+    end
+
+    before do
```

Comment:
Imho `before` block should be defined in the beginning of appropriate `describe` to be more readable.